docs/docs_settings.py

```diff
@@ -7,17 +7,10 @@
 
 import os
 
-from lms.envs.devstack import *  # lint-amnesty, pylint: disable=wildcard-import
-from cms.envs.devstack import (  # lint-amnesty, pylint: disable=unused-import
-    ADVANCED_PROBLEM_TYPES,
-    COURSE_IMPORT_EXPORT_STORAGE,
-    GIT_EXPORT_DEFAULT_IDENT,
-    LIBRARY_AUTHORING_MICROFRONTEND_URL,
-    SCRAPE_YOUTUBE_THUMBNAILS_JOB_QUEUE,
-    VIDEO_TRANSCRIPT_MIGRATIONS_JOB_QUEUE,
-    UPDATE_SEARCH_INDEX_JOB_QUEUE,
-)
-
+if os.environ['EDX_PLATFORM_SETTINGS'] == 'devstack_docker':
```

Comment:
why do we need this setting check here? Mention any reference for this change if available.

Reply:
May be it was removed in master branch but when I rebased I by mistake added them again. I'll remove these

Reply:
Looking at these again, these should be kept as per the reason mentioned in the doc strings in the file.